EncoreTickets.SDK/Utilities/RestClientWrapper/RestClientWrapper.cs

```diff
@@ -188,14 +178,9 @@ private void SetRequestSerializer(IRestRequest request, RestClientParameters res
             }
         }
 
-        private bool IsGoodResponse(IRestResponse response)
+        private bool ShouldRequestBeRepeated(IRestResponse response)
```

Comment:
Why is this change made? Also I don't think this is a good name here because you are using it to describe the response that you want to handle

Reply:
We need to repeat only those requests that have responses with a server error because in such cases we have a chance that something will change the next time we try. If we get another error, it shows that something is wrong with the request, and repeating the same request should not help.
I don’t understand why this name is not suitable, because it clearly describes what I want to check. This method currently checks if the response was with a server error, but it may contain more conditions and the purpose of the method will not change

Reply:
The reason I don't really like the name is because you are only using it in the OrResult where you are supposed to describe a response condition which should cause the request to be repeated. In such case the old method name seems better suited for that. And if you ever try to use this method anywhere else or move it to some RestResponseExtension, then its name would make even less sense

Reply:
This is a private method used for certain purposes, so I do not think that this method will be used in some places outside this class in the future. And in this case, the method name looks good to me. And also I think that the old name is not suitable, because it is strange to call the IsGoodResponse method when the response may be with a client error, in which case the method will return true

Reply:
Ok, let's leave it like that for now then